tests/storage/golden_image/test_golden_image.py

```
@@ -6,24 +6,38 @@
-RHEL_IMAGE_SIZE = RHEL_LATEST["dv_size"]
 
 
-DV_PARAM = {
-    "dv_name": "golden-image-dv",
-    "image": LATEST_RHEL_IMAGE,
-    "dv_size": RHEL_IMAGE_SIZE,
-    "storage_class": py_config["default_storage_class"],
-}
+@pytest.fixture(scope="module")
+def golden_image_dv_from_fedora_datasource_scope_module(
+    request,
```

Ruff (0.15.15)

[warning] Unused function argument: `request`

(ARG001)